src/Startup.cs

```diff
@@ -49,10 +49,14 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddSingleton<SearchUiConfig, SearchUiConfig>();
 
+            services.AddHttpClient<IHttpClient, HttpClientWrapper>(x => x.Timeout = TimeSpan.FromSeconds(15))
```

Comment:
do we really want the timeout to be that high?

Reply:
Briefly testing locally, there are requests that take ~9 seconds...